### DISCUSSION TITLE
Error when attempting to build ipa

### DISCUSSION THREAD

**Discussion by ghost** [SoCuul/SCInsta#45] [Support] [answered]
Hey there. I've installed the pre-requisites and ios 14.5 frameworks, but I get this error when running the build script (on macos).

```
Makefile:5: /makefiles/common.mk: No such file or directory
Makefile:17: /tweak.mk: No such file or directory
make: *** No rule to make target `/tweak.mk'.  Stop.
```

common.mk and tweak.mk are definitely in the ~/theos folder so not sure what I'm doing wrong.

**Comment by SoCuul** [comment, marked as answer]
Did you make sure to add theos to your system path?

**Reply by ghost** [reply]
I just looked into the makefile and realised there was reference to $(THEOS)... running `export THEOS=~/theos` certainly did fix my issue. Thanks!
